src/homelogstore/log_store.cpp

```diff
@@ -388,6 +391,16 @@ sisl::status_response HomeLogStore::get_status(const sisl::status_request& reque
     response.json["truncation_parallel_to_writes?"] = m_safe_truncation_boundary.active_writes_not_part_of_truncation;
     response.json["logstore_records"] = m_records.get_status(request.verbose_level);
     response.json["logstore_sb_first_lsn"] = m_logdev.m_logdev_meta.store_superblk(m_store_id).m_first_seq_num;
+
+    if (request.json.contains("type") && request.json["type"] == "logstore_record") {
```

Comment:
I didn't find the place where we create_object with this type `logstore_record`, is this expected?

Reply:
We didnt create similar to btree nodes, so user can specify all records for a logstore name=HomeLogStore_0&type=logstore_record